src/platform/library/include/platform/lib/memory.h

```diff
@@ -65,7 +65,7 @@ static inline uint32_t arch_get_stack_size(void)
 
 #define SRAM_BANK_SIZE	0x10000
 #define LP_SRAM_SIZE SRAM_BANK_SIZE
-#define HP_SRAM_SIZE (SRAM_BANK_SIZE * 47)
+#define HP_SRAM_SIZE (SRAM_BANK_SIZE * 47)	/* align to Intel cAVS CNL platform */
```

Comment:
why are you adding this comment in the build for the host? The value may end up being chosen arbitrarily so we can build all components for the test bench at once.